README.md

```diff
@@ -1,37 +1,464 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/prometheus-operator-app.svg?style=shield)](https://circleci.com/gh/giantswarm/prometheus-operator-app)
+# Prometheus Operator App
 
-# prometheus-operator-app
+[![CircleCI](https://circleci.com/gh/giantswarm/promtail-app.svg?style=shield)](https://circleci.com/gh/giantswarm/prometheus-operator-app)
 
-Giant Swarm offers a Prometheus Operator Managed App which can be installed in
-workload clusters. Here we define the Prometheus chart with its templates and
-default configuration.
+Giant Swarm offers Prometheus Operator as a [managed app](https://docs.giantswarm.io/changes/managed-apps/) which can be installed in any clusters.
+
+**Table of Contents:**
+
+- [Prometheus Operator App](#prometheus-operator-app)
```

Comment:
Weird to have the table of contents after first paragraphe.
If it's what you want, I suggest to remove the first link in the table because it references the above paragraphe and it's weird !

Reply:
Fixed